lib/cartopy/tests/test_coding_standards.py

```diff
@@ -29,12 +26,11 @@
 LICENSE_RE_PATTERN = re.escape(LICENSE_TEMPLATE)
 LICENSE_RE = re.compile(SHEBANG_PATTERN + LICENSE_RE_PATTERN, re.MULTILINE)
 
-
-# Guess cartopy repo directory of cartopy - realpath is used to mitigate
-# against Python finding the cartopy package via a symlink.
-CARTOPY_DIR = Path(cartopy.__file__).parent.resolve()
-REPO_DIR = Path(os.getenv('CARTOPY_GIT_DIR',
-                          CARTOPY_DIR.parent.parent))
+# Get repo directory for cartopy (https://stackoverflow.com/a/53675112)
+REPO_DIR = subprocess.Popen(['git', 'rev-parse', '--show-toplevel'],
+                            stdout=subprocess.PIPE
+                            ).communicate()[0].rstrip().decode('utf-8')
+REPO_DIR = Path(REPO_DIR)
```

Comment:
Do we need to keep supporting `CARTOPY_GIT_DIR`?
```suggestion
REPO_DIR = Path(os.getenv("CARTOPY_GIT_DIR", REPO_DIR))
```